src/regression_tests/utils.ts

```diff
@@ -0,0 +1,183 @@
+
+  return user;
+};
+
+// removeRegisteredPaymail soft deletes paymail from the SPV Wallet.
+export const removeRegisteredPaymail = async function (
+  paymail: string,
+  instanceURL: string,
+  adminXPriv: string,
+): Promise<void> {
+  const adminClient = new SpvWalletClient(instanceURL, { adminKey: adminXPriv, xPriv: adminXPriv });
+  await adminClient.AdminDeletePaymail(paymail);
+};
+
+// getBalance retrieves the balance from the SPV Wallet.
+export const getBalance = async function (fromInstance: string, fromXPriv: string): Promise<number> {
+  const client = new SpvWalletClient(fromInstance, { xPriv: fromXPriv });
+
+  const xpubInfo = await client.GetXPub();
+
+  return Number(xpubInfo.current_balance);
+};
+
+// getTransactions retrieves the transactions from the SPV Wallet.
+export const getTransactions = async function (fromInstance: string, fromXPriv: string): Promise<Txs> {
+  const client = new SpvWalletClient(fromInstance, { xPriv: fromXPriv });
+
+  const metadata = new Map<string, any>();
+  const conditions: TransactionFilter = {};
+  const queryParams: QueryParams = {};
+
+  const txs = await client.GetTransactions(conditions, metadata, queryParams);
+
+  return txs;
+};
+
+// sendFunds sends funds from one paymail to another
+export const sendFunds = async function (
+  fromInstance: string,
+  fromXPriv: string,
+  toPaymail: string,
+  howMuch: number,
+): Promise<Tx> {
+  const client = new SpvWalletClient(fromInstance, { xPriv: fromXPriv });
+
+  const balance = await getBalance(fromInstance, fromXPriv);
+  if (balance < howMuch) {
+    throw new Error(`Insufficient funds: ${balance}`);
+  }
+
+  const recipients: Recipients = [{ to: toPaymail, satoshis: howMuch }];
+  const metadata = { description: 'regression-test' };
+
+  const transaction = await client.SendToRecipients(recipients, metadata);
+
+  const tx = await client.GetTransaction(transaction.id);
+
+  return tx;
+};
+
+// preparePaymail prepares the paymail address by combining the alias and domain.
+export const preparePaymail = function (paymailAlias: string, domain: string): string {
+  if (isValidURL(domain)) {
+    const splitedDomain = domain.split('//');
+    domain = splitedDomain[1];
+  }
+  const url = paymailAlias + AT_SIGN + domain;
+  return url;
+};
+
+// addPrefixIfNeeded adds the HTTPS prefix to the URL if it is missing.
+export const addPrefixIfNeeded = function (url: string): string {
+  if (!isValidURL(url)) {
+    return DOMAIN_PREFIX + url;
+  }
+  return url;
+};
+
+// isValidURL validates the URL if it has http or https prefix.
+export const isValidURL = function (rawURL: string): boolean {
+  return EXPLICIT_HTTP_URL_REGEX.test(rawURL);
+};
+
+// addApiVersionIfNeeded adds api version if it's not present in the url
+const addApiVersionIfNeeded = function (url: string, apiVersion: string): string {
+  const apiVersionWithSlash = '/' + apiVersion;
+  if (url.includes(apiVersionWithSlash)) {
+    return url;
+  } else {
```

Comment:
No `else` needed because the return in the `if` block above breaks the function execution.

Reply:
Ah you're right, removed else.